Backend/wrappers/standardAPI.py

```diff
@@ -1,24 +1,43 @@
-import tweepy
-
 import wrappers.credentials as creds
 
-
 class standardAPI:
 
     def __init__(self):
-        self.api = creds.create()
-
-
-    def retrieve_trends(self, woeid, num):
-        query = self.api.trends_place(woeid)
-        top_trends  = self.query_transform(query, woeid, num)
-        return top_trends
+        self.tweepy = creds.getStandardTweepyTwitterCreds()
+        self.pytwit = creds.getStandardPythonTwitterCreds()
 
+    def retrieve_trends(self, woeid):
+        query = self.tweepy.trends_place(woeid)
+        # top_trends  = self.query_transform(query, woeid, num)
+        return query
 
-    def getTrendsClose(self, lat, long, num):
-        query = self.api.trends_closest(lat, long)
+    def get_nearby_location(self, lat, long):
+        query = self.tweepy.trends_closest(lat, long)
         return query
-    
+
+    # Retrieve tweets using the Twitter 7 Days STANDARD API endpoint
+    # Args:
+    #   keyword: query string
+    #   since: the date string of where to start searching tweets
+    #   until : end date string of where to stop searching tweets
+    #   num: number of tweets to return
+    #   location: optional location object
+    def getTweets(self, keyword, since, until, num, location=None):
+
+        if location is not None:
+            radius = "100mi"
+            geocode = "{},{},{}".format(location.min_latitude, location.min_longitude, radius)
+            tweets = self.pytwit.GetSearch(term=keyword,
+                                           geocode=geocode,
+                                           since=since, until=until,
+                                           result_type='top', count=num)
+
+            return tweets
+        else:
+            tweets = self.pytwit.GetSearch(term=keyword, since=since, until=until,
```

Comment:
I actually threw in another dependency because I thought Tweepy didn't support the search. I guess i looked in the wrong place and missed it. There was no reason to the added dependency other than being a quick solution to a problem I was having sorry~. 
If you want I can remove this dependency and just use Tweepy for the next sprint